plans/add_replica.pp

```diff
@@ -90,10 +90,10 @@
   }
 
   run_plan('peadm::util::update_classification', $primary_target,
-    server_a_host                    => $replica_avail_group_letter ? { 'A' => $replica_host, default => undef },
-    server_b_host                    => $replica_avail_group_letter ? { 'B' => $replica_host, default => undef },
-    internal_compiler_a_pool_address => $replica_avail_group_letter ? { 'A' => $replica_host, default => undef },
-    internal_compiler_b_pool_address => $replica_avail_group_letter ? { 'B' => $replica_host, default => undef },
+    server_a_host                    => $replica_avail_group_letter ? { 'A' => $replica_target.peadm::certname(), default => undef },
```

Comment:
At the top this plan still has:

```
# This plan is still in development and currently considered experimental.
#
# @api private
```

is that still the case?

Reply:
This is being removed in the below PR:

https://github.com/puppetlabs/puppetlabs-peadm/pull/478